src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -37,10 +40,10 @@ public class Shooter extends SubsystemBase implements Logged {
   private final SimpleMotorFeedforward topSimFeedforward;
   private final SimpleMotorFeedforward bottomSimFeedforward;
 
-  @Log private double topSetpoint;
-  @Log private double bottomSetpoint;
-  @Log private double topShoterVelocity;
-  @Log private double bottomShoterVelocity;
+  @Log private Measure<Velocity<Angle>> topSetpoint;
```

Comment:
Monologue can only log primitive data types. See wiki for details:
https://github.com/shueja/Monologue/wiki

You'll need to have both the Measure type object and a double that is just used for logging.